Dockerfile

```diff
@@ -1,51 +1,45 @@
-FROM ubuntu:22.04
+FROM debian:bookworm
 
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Install system packages then clean up to minimize image size
 RUN apt-get update \
- && apt-get install --no-install-recommends -y \
-      build-essential \
-      curl \
-      default-jre-headless \
-      file \
-      git-core \
-      gpg-agent \
-      libarchive-dev \
-      libffi-dev \
-      libgd-dev \
-      libpq-dev \
-      libsasl2-dev \
-      libvips-dev \
-      libxml2-dev \
-      libxslt1-dev \
-      libyaml-dev \
-      locales \
-      postgresql-client \
-      ruby \
-      ruby-dev \
-      ruby-bundler \
-      software-properties-common \
-      tzdata \
-      unzip
-
-# Install Node.js 18 and npm
-RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
- && apt-get install -y nodejs
+  && apt-get install --no-install-recommends -y \
+  build-essential \
+  curl \
+  default-jre-headless \
+  file \
+  git-core \
+  gpg-agent \
+  libarchive-dev \
+  libffi-dev \
+  libgd-dev \
+  libpq-dev \
+  libsasl2-dev \
+  libvips-dev \
+  libxml2-dev \
+  libxslt1-dev \
+  libyaml-dev \
+  locales \
+  postgresql-client \
+  ruby \
+  ruby-dev \
+  ruby-bundler \
+  software-properties-common \
+  tzdata \
+  unzip \
+  nodejs \
+  npm \
+  osmosis
 
 # Install yarn globally
-RUN npm install --global yarn \
- # We can't use snap packages for firefox inside a container, so we need to get firefox+geckodriver elsewhere
- && add-apt-repository -y ppa:mozillateam/ppa \
- && echo "Package: *\nPin: release o=LP-PPA-mozillateam\nPin-Priority: 1001" > /etc/apt/preferences.d/mozilla-firefox \
- && apt-get install --no-install-recommends -y \
-      firefox-geckodriver \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
-
-# Install compatible Osmosis to help users import sample data in a new instance
-RUN curl -OL https://github.com/openstreetmap/osmosis/releases/download/0.47.2/osmosis-0.47.2.tgz \
- && tar -C /usr/local -xzf osmosis-0.47.2.tgz
+RUN npm install --global yarn
+
+# Install geckodriver and firefox
+RUN apt-get update                             \
+  && apt-get install -y --no-install-recommends \
+  ca-certificates curl firefox-esr           \
+  && curl -L https://github.com/mozilla/geckodriver/releases/download/v0.35.0/geckodriver-v0.35.0-linux64.tar.gz | tar xz -C /usr/local/bin
```

Comment:
I think we might be able to drop this - it seems that the ruby webdrivers gem is able to download this itself as needed now.

Reply:
Tests are passing without downloading "curl -L https://github.com/mozilla/geckodriver/releases/..." first. So yes, installing geckodriver could be dropped as well.

Reply:
You're right. Tests are passing without it. I removed it accordingly. Thank you @tomhughes @mmd-osm